config/config-builder/src/validator_config.rs

```diff
@@ -262,6 +262,7 @@ impl ValidatorConfig {
                 "in-memory" => SecureBackend::InMemoryStorage,
                 "on-disk" => safety_rules_config.backend.clone(),
                 "vault" => SecureBackend::Vault(VaultConfig {
+                    certificate: None,
```

Comment:
Can we include "CA" in the name (Vault can do client certificate auth, so we should distinguish this from a client cert). `ca_certificate` or `ca_cert` etc.

Reply:
makes sense, renamed to server_certificate

Reply:
Well, it's not the server certificate, it's the CA certificate.
It may actually be nice to use the same environment variables that Vault uses (`VAULT_CACERT` for this) because then users can set it once and Terraform, `vault` and this will all pick it up.

Reply:
(Although I suppose this doesn't work when the Association also uses Vault and there are two CA certs in play.)

Reply:
Yeah, not going to touch environmental variables in this case... as convenient as they are, they create nightmares in this case.

I renamed to ce_certificate. You are right, in my case, it was a "server_certificate" but it generalizes to a "ca_certificate" :).